### PR TITLE
Feat/dockerfile

### PR DESCRIPTION
**Pull request by Disturbing**
This works and has been tested on `beta.kintohub.com`

Requires **NO BUILD COMMAND** Simply add to the folder `/app/website/kintohub-docs` as the output path.

---
This change is [Reviewable](https://reviewable.io/reviews/kintohub/kintohub-docs/160)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /app
+COPY ./website /app
+COPY ./docs /docs
+
+RUN npm install
+RUN npm run build
+
+ENTRYPOINT ["npm", "start"]
+
```
Comment (bappr) [inline Dockerfile]:
```
FROM node:8.11.4

WORKDIR /app

COPY package.json .

RUN npm install

COPY ./website .
COPY ./docs /docs

RUN npm run build

ENTRYPOINT ["npm", "start"]
```

Comment (Disturbing) [reply Dockerfile]: The package.json is in ./website/package.json - did you test this or is it sudo?

Comment (bappr) [reply Dockerfile]: Sorry updated. Did not test it, just added the improvement.

**Comment by Disturbing** [comment]
Dockerfile no longer needed, can use nodejs non alpine :).

**Pull request closed without merging**